mms/README.md

```diff
@@ -13,6 +13,28 @@ cd mms
 docker-compose kill
 docker-compose rm -f
 ```
+### Importing the MMS keyspaces and data automatically 
+
+Add the following argument under ```environment``` for ```scylla-node1:``` in docker-compose.yml:
+
+```
+  - IMPORT=IMPORT
+```
```

Comment:
I do not understand this part. Why is it needed? 
Can this line be added to docker-compose.yml for good?

Reply:
@tzach  This will automatically import the keyspaces and data automatically if set. Useful for people wanting to get started later or if we use this for demos or tradeshows.

Reply:
To which of the example is it relevant?

Reply:
It will be relevant going forward, good to add now. People can jump in later. 

I updated the monitoring README and added a git submodule. I had to modify the scripts a bit to make it work with the MMS.